app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav-related.scss

```diff
@@ -15,6 +15,22 @@
   padding: 0;
 }
 
+.gem-c-step-nav-related-singular {
+  margin-bottom: $gutter-one-third + 3;
+}
+
+.gem-c-step-nav-related-singular .gem-c-step-nav-related__heading {
+  @include _core-font-generator(19px, 19px, 19px, 1.4, 1.4, false, bold);
```

Comment:
I've noticed you use the core-font-generator mixin a lot - are you aware you can use `@include core-19` or `@include bold-19` instead? ([see here](https://github.com/alphagov/govuk_frontend_toolkit/blob/452f82e7538d7f5b8f73ee7d81abea9bc857af13/stylesheets/_typography.scss#L92)). Obviously if you want to use custom line heights etc. then that's fine, although we should  use the standard mixins where reasonable, just for consistency and ease of maintenance.

Reply:
The reason why I used the core-font-generator is because we needed the related size to be consistent with the step title. https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav.scss#L62

If I use bold-19 or core-19, the font size variable between mobile and desktop.